packages/shared/src/graphql/submitArticle.ts

```diff
@@ -0,0 +1,49 @@
+import { gql } from 'graphql-request';
+import { Post } from './posts';
+
+interface Submission {
+  id: string;
+  url: string;
+  userId: string;
+  createdAt: Date;
+  status: string;
+  reason: string;
+}
+
+export interface SubmitArticleResposne {
+  result: 'succeed' | 'exists' | 'reject';
```

Comment:
Don't know if it's too late but it should be `rejected` if we want to keep it consistent. Minor thing

Reply:
Changed it in the API as well :)